app/core/rate_limiting/schemas.py

```
@@ -0,0 +1,77 @@
+    
+    flash_stats: RateLimitMetadata = Field(description="Gemini 2.5 Flash usage stats")
+    pro_stats: RateLimitMetadata = Field(description="Gemini 2.5 Pro usage stats")
+    
+    flash_circuit: CircuitBreakerStatus = Field(description="Flash circuit breaker status")
+    pro_circuit: CircuitBreakerStatus = Field(description="Pro circuit breaker status")
+    
+    total_requests_today: int = Field(description="Total requests across all models today")
+    total_requests_this_minute: int = Field(description="Total requests across all models this minute")
+    
+    uptime_percentage: float = Field(description="Service uptime percentage")
+    last_updated: datetime = Field(default_factory=datetime.utcnow, description="When stats were last updated")\n    \n    class Config:\n        orm_mode = True
```

Ruff (0.11.9)

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

Multiple statements on one line (colon)

(E701)